classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -54,6 +54,8 @@
  *   required: Adds an asterisk and a .required class to the element's label
  */
 
+define('INSERT_TAG_VARIABLE_TYPE_PLAIN_TEXT', 'PLAIN_TEXT');
```

Comment:
Try exploring `$.pkp.cons` from the JS console -- it should be there. I haven't used this facility in quite a while, though. It's implemented in PKPTemplateManager (look for `getExposedConstants`).